formbuilder/routes.js

```diff
@@ -17,7 +17,7 @@ import Check from "./components/Check";
 
 const common = {
   notifications: NotificationContainer,
-  fieldList: FieldListContainer,
+  sidebarComponent: FieldListContainer,
```

Comment:
The `FieldListContainer` container still remains a field list container, so I don't think we should change its name :)